website/servers/server.php

```diff
@@ -185,8 +260,10 @@
 				<h1>Edit server</h1>
 				<div class="formrow">
 					<p><label class="label">Location: <input class="textbox" type="text" name="loc" placeholder="http://<?php echo $entry['server']; if ($entry['port'] != 8000) echo '-',$entry['port']; ?>.psim.us/" size="60" /><em>(you can leave off http:// etc if you want)</em></label></p>
-					<p><label class="label">Owner: <input class="textbox" type="text" name="owner" value="<?php echo $entry['owner'] ?? '(none)'; ?>" size="60" /><em>(separate multiple owners by commas)</em></label></p>
-					<label class="label">Configuration:</label><?php
+					<p><label class="label">Owner: <strong><?php echo $entry['owner'] ?? '(none)'; ?></strong></label></p>
```

Comment:
Why isn't the owner editable anymore? PS admins like me should be able to edit it without having to SSH in...

Reply:
Uh, it definitely still is. I think the button just got removed somehow.

Reply:
Fixed.